### PR TITLE
update xdr docs

### PR DESCRIPTION
**Pull request by OmriItzhak**
## Status
Ready/In Progress/In Hold (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-dc.paloaltonetworks.com/browse/CIAC-9678)

## Description
xdr lite playbooks Updated the pack Documentation.

## Screenshots
Paste here any images that will help the reviewer

### CODE REVIEW

File: content-repo/extra-docs/packs/palo-alto-networks-cortex-xdr---investigation-and-response.md [hunk 1]
```diff
@@ -46,6 +46,9 @@ If the verdict is set to benign, the playbook will close the incident.
 
 As part of this playbook, you'll receive a comprehensive layout that presents incident details, analysis, investigation findings, and the final verdict. Additionally, the layout offers convenient remediation buttons for quicker manual actions.
 
+To utilize this playbook as the default for handling XDR incidents, the classifier should be empty, and the selected incident type should be `Cortex XDR - Lite`.
+The selected Mapper (incoming) should be `XDR - Incoming Mapper`, And the selected Mapper (outgoing) should be Cortex `XDR - Outgoing Mapper`.
```
Comment (ShirleyDenkberg) [inline content-repo/extra-docs/packs/palo-alto-networks-cortex-xdr---investigation-and-response.md:50]:
```suggestion
The selected Mapper (incoming) should be `XDR - Incoming Mapper`, and the selected Mapper (outgoing) should be Cortex `XDR - Outgoing Mapper`.
```

File: content-repo/extra-docs/packs/palo-alto-networks-cortex-xdr---investigation-and-response.md [hunk 2]
```diff
@@ -86,6 +89,9 @@ If this was a port scan alert, the analyst will manually block the ports used fo
 
 After the remediation, if there are no new alerts, the playbook stops the alert sync and closes the XDR incident and investigation. 
 
+To utilize this playbook for handling XDR incidents, the classifier that should be selected is `Cortex XDR - Classifier`.
+The selected Mapper (incoming) should be `XDR - Incoming Mapper`, And the selected Mapper (outgoing) should be Cortex `XDR - Outgoing Mapper`.
```
Comment (ShirleyDenkberg) [inline content-repo/extra-docs/packs/palo-alto-networks-cortex-xdr---investigation-and-response.md:93]:
```suggestion
The selected Mapper (incoming) should be `XDR - Incoming Mapper`, and the selected Mapper (outgoing) should be Cortex `XDR - Outgoing Mapper`.
```

Comment (ShirleyDenkberg) [inline content-repo/extra-docs/packs/palo-alto-networks-cortex-xdr---investigation-and-response.md:93]:
```suggestion
The selected Mapper (incoming) should be `XDR - Incoming Mapper`, and the selected Mapper (outgoing) should be Cortex `XDR - Outgoing Mapper`.
```

**Comment by ShirleyDenkberg** [comment]
@altmannyarden Doc review completed.

**Review by altmannyarden — APPROVED** [review]

**Pull request merged**